toolchain/parse/state.def

```diff
@@ -670,35 +670,31 @@ CARBON_PARSE_STATE(FunctionSignatureFinish)
 //   (state done)
 CARBON_PARSE_STATE(FunctionDefinitionFinish)
 
-// Handles `export`.
-//
-// export import library "libname";
-// ^~~~~~
-//   1. Import
+// Handles `export <name>`.
 //
 // export Name;
 // ^~~~~~
```

Comment:
It looks like `HandleExportName` expects the `export` to already have been consumed by `HandleDeclScopeLoop`, so should this `^~~~~~` be moved to the comment on `DeclScopeLoop`?

Reply:
Done